src/Http/Validator/AllOf.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Utopia\Http\Validator;
+
+use Utopia\Http\Validator;
+
+/**
+ * Ensure all of the validators from a list passed the check
+ *
+ * @package Utopia\Validator
+ */
+class AllOf extends Validator
+{
+    /**
+     * @param array<Validator> $validators
+     */
+    public function __construct(protected array $validators, protected string $type = self::TYPE_MIXED)
+    {
+    }
+
+    /**
+     * Get Description
+     *
+     * Returns validator description
+     *
+     * @return string
+     */
+    public function getDescription(): string
+    {
+        $description = '';
+        foreach ($this->validators as $key => $rule) {
+            $description .= ++$key . '. ' . $rule->getDescription() . " \n";
+        }
+
+        return $description;
```

Comment:
This might be misleading, most validators will return an error message from `getDescription` even if they were valid.

E.g. `AllOf(Text(10), Json)` with value `'hello'` will output:

```
1. Value must be a valid string and no longer than 10 chars
2. Value must be a valid JSON string
```

Let's make it so the message only contains errors from validators that failed validation